src/models/activityReport.js

```diff
@@ -28,6 +28,8 @@ export default (sequelize, DataTypes) => {
       ActivityReport.belongsTo(models.Region, { foreignKey: 'regionId', as: 'region' });
       ActivityReport.hasMany(models.File, { foreignKey: 'activityReportId', as: 'attachments' });
       ActivityReport.hasMany(models.File, { foreignKey: 'activityReportId', as: 'otherResources' });
+      ActivityReport.hasMany(models.NextStep, { foreignKey: 'activityReportId', as: 'specialistNotes' });
+      ActivityReport.hasMany(models.NextStep, { foreignKey: 'activityReportId', as: 'granteeNotes' });
```

Comment:
I think this is fine as is now, and I don't know how to make it better, but I wish we could apply the where clause at this point instead of forcing the caller to remember to add `WHERE noteType === "GRANTEE"`. Might be worth looking into if there is a better way before this becomes a pattern.